iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -74,10 +89,16 @@ class ThreadInitByFunc : public Continuation
 } Thread_Init_Func;
 }
 
+void *
+ThreadAffinityInitializer::alloc_hugepage_stack(size_t stacksize)
```

Comment:
This naming seems a little unfortunate. alloc_stack_memory() or something instead?